pkg/agent/controller/egress/egress_controller.go

```diff
@@ -1118,39 +1117,42 @@ func (c *EgressController) syncEgress(egressName string) error {
 	}()
 
 	egressIP := net.ParseIP(eState.egressIP)
-	// Install SNAT flows for desired Pods.
-	for pod := range pods {
-		eState.pods.Insert(pod)
-		stalePods.Delete(pod)
+	egress, err = c.egressLister.Get(egressName)
```

Comment:
lint check failed, err is not used, address lint issue

Reply:
done